documentation/cluster-controller.adoc

```diff
@@ -74,6 +75,8 @@ the related ConfigMap :
  Removing this field means having no metrics exposed.
 * `zookeeper-metrics-config`: a JSON string representing the JMX exporter configuration for exposing metrics from Zookeeper nodes.
  Removing this field means having no metrics exposed.
+* `topic-controller-config`: a JSON string representing the topic controller configuration. See related section for that.
+More info about the topic controller in the related <<Topic Controller>> documentation page.
```

Comment:
* `topic-controller-config`: a JSON string representing the topic controller configuration. See the  <<Topic Controller>> documentation for further details.